### PR TITLE
Add admin guide for glusterfs-coreutils

### PR DESCRIPTION
**Pull request by anoopcs9**
General overview on how to install and use glusterfs-coreutils package.

closes gluster/glusterfs-coreutils#9

Signed-off-by: Anoop C S anoopcs@redhat.com

### CODE REVIEW

File: Administrator Guide/GlusterFS Coreutils.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+##### For fedora
+Use dnf/yum to install glusterfs-coreutils:
+
+    dnf install glusterfs-coreutils
+OR
+
+    yum install glusterfs-coreutils
+
+RPMs are also available at <http://download.gluster.org/gluster/glusterfs-coreutils/>
```
Comment (prashanthpai) [inline Administrator Guide/GlusterFS Coreutils.md]: Dead link. I guess this will be fixed once RPMs are generated ?

Comment (anoopcs9) [inline Administrator Guide/GlusterFS Coreutils.md]: Yes.

**Comment by anoopcs9** [comment]
I will re-open this PR as soon as https://github.com/gluster/glusterdocs/pull/116 gets merged.

**Comment by anoopcs9** [comment]
Closing this due to mismatch in commits along with this PR.

**Pull request closed without merging**